src/librustdoc/html/static/rustdoc.css

```diff
@@ -1578,7 +1578,7 @@ h4 > .notable-traits {
 	}
 }
 
-@media (max-width: 416px) {
+@media (max-width: 464px) {
```

Comment:
Feels like a hacky solution to me, what if the width is slightly smaller or larger? Ideally we would want something to do with dynamic width like `calc` but that would also make stuff more complicated.

Reply:
It actually depends on the min width of the search input mostly. ;)

Reply:
Other than that, on mobile, once you click on the menu, the body jumps off, that behavior looks weird.

Reply:
Do you mean the body jumping down when you click the hamburger? That appears to be a nightly regression, it doesn't happen on the stable docs.